doc/tech/approach.md

```diff
@@ -97,3 +97,101 @@ As with most CNTT subsystems, responsibility for Objectives, Requirements, Guide
+
+Cloud and Telco networking are layered, and it is very important to keep the layering dependences low to enable security, separation and portability in between multiple implementations.
+
+These explanations for Layering and Concepts are likely too rich and verbose for a CNTT document entry, but it is important that we first understand and agree on these concepts before we start entering text that could be misunderstood.
+
+### Underlay and Overlay Networking concepts
+
+The ETSI NFV model divide networking in an Underlay and an Overlay Network layer. The purpose with this layering is to ensure separation of the SW Virtualization tenants Overlay Networks from each other, whilst allowing the traffic to flow on the shared Underlay Network in between all Ethernet connected HW units.
+
+The Overlay Networking separation is often done through encapsulation e.g. through VxLAN on the Underlay Networks e.g. based on L2 (VLAN) or L3 (IP) networks.
+
+In some instances, the SW Virtualization Tenants can bypass the Overlay Networking encapsulation to achieve better performance or network visibility/control. A common method to bypass the Overlay Networking encapsulation is the usage of SR-IOV that effectively hands up the NIC Physical and Virtual Functions on the NIC to the SW Virtualization Layer and Tenants. In these cases, the Underlay Networking must handle the separation e.g. through a Virtual Termination End Point (VTEP) that encapsulate the Overlay Network traffic.
+
+### Software Defined Networking control concepts of the Underlay Networking
+
+VTEP could be manually provisioned in the Underlay Networking or be automated and controlled through a Software Defined Networking interfaces to the Underlay Networking in the HW Infrastructure Layer. Due to the many different facets of Software Defined Networking we will here denote them SDN Underlay (SDNu).
+
+When there are multiple simultaneous SW Virtualization Layers on the same HW Infrastructure, there is a need to ensure Underlay networking separation in the HW Infrastructure Layer. This separation can be done manually through provisioning of a statically configured separation of the Underlay networking in the HW Infrastructure Layer. A better and more agile usage of the HW Infrastructure is to have an authoritative SDN provisioning controller function (here denoted SDNuP) that can be controlled through an automation interface from a HW Infrastructure Orchestrator. The main tasks for the SDNuP are to discover and establish the Underlay resources and then ensure separation of the shared HW Infrastructure Underlay networking resources.
+
+Multiple instances of a Containerized Virtualization Layer (CaaS) running on an Infrastructure as a Service (IaaS) Virtualization Layer could make use of the IaaS layer to handle the required Underlay Networking separation. In these cases, also the IaaS Virtualization Infrastructure Manager (VIM) could include a SDNu control interface enabling automation.
```

Comment:
New term for "Virtualization Infrastructure Manager (VIM)" is "Cloud Infrastructure Manager"

Reply:
No, "NFVI" is now "Cloud Infrastructure" in CNTT language to better include container infrastructure, but I can not see that this extend to a Virtualization Infrastructure Manager. ETSI states that there can be multiple Virtualization Infrastructure Managers and to what I see that fits well with our approach with a HW Infrastructure Manager and several SW Virtualization Managers, that could be of different types e.g. managing an IaaS or managing a CaaS.

Reply:
@TFredberg Please see RM 5.1 Cloud Infrastructure Software profile description (under Figure 5-1) where the following:

"For a host (compute node or physical server), the virtualisation layer is an abstraction layer between hardware components (compute, storage, and network resources) and virtual resources allocated to VNFC. Each VNFC generally maps 1:1 against a single VM or a single container/pod. **Figure 5-2** represents the virtual resources (virtual compute, virtual network, and virtual storage) allocated to VNFC and managed by the **VIM**."

has been changed to (highlighting mine):
"For a host (compute node or physical server), the virtualisation layer is an abstraction layer between hardware components (compute, storage, and network resources) and virtual resources allocated to a VM or a Pod. **Figure 5-2** represents the virtual resources (virtual compute, virtual network, and virtual storage) allocated to a VM or a Pod and managed by the **Cloud Infrastructure Manager**."

Reply:
@pgoyal01 This text is not under my editorial pen and will likely be superseeded by @walterkozlowski text entered in #1325 .
However in response to your suggested change I do not agree on that we shall change all ETSI NFV terminology such as a VIM into a Cloud Infrastructure Manager.
In my text below in the section about Layering and Concepts, I am trying to explain that the ETSI NFV Management and Orchestration view of having multiple VIMs fits nicely for our view of the need to managing different aspects and different instances of the Cloud Infrastructure. WIth that view, we should not call all of these an Cloud Infrastructure Manager, but rather denominate them according to their roles e.g. HW Infrastructure Manager and maybe, if we so decide Container Infrastructure Manager (for CaaS) and Virtialization Infrastructure Manager (for IaaS). In ETSI NFV it is common that OpenStack oriented systems tend to claim that "the VIM" is the OpenStack manager, so it might be good to stay with at least the acronym VIM for RA-1 type of SW Virtualization Managers.

Reply:
@TFredberg I was and remain not keen on changing terminology (I believe VNF and NFV cover all forms of virtualisation).  Given that the terminology is being changed in all documents, using different terms may be confusing to the reader and hence the ask.

Reply:
@pgoyal01 Do you think it would be a good strategy to take this up with the Networking Focus Group (that assigned these PR´s) on Monday and then have it taken up on RM on Wednesday for potential further discussions on the TSC?

Reply:
@TFredberg excellent suggestion. Thanks

Reply:
@kedmison as dicussed in RM call :) , different context to use different terminologies

Reply:
@pgoyal01 @ASawwaf @TFredberg @kedmison Yes, let us discuss on Monday NFG meeting.  My view is that in the context of the Networking content we are working on we should refer to the ETSI standard terminology, e.g.  because we are discussing gaps between ETSI model and our needs. We can make a note that VIM maps to Cloud Infrastructure Manger.  Similarly with other terminology issues of this kind. @TFredberg @pgoyal01 would you agree with such position?  At least for now, until we take some stand on it in TSC.  We do not want to lose our energy at this initial stage when defining Networking Fabric genral approach, scope and HL model.

Reply:
@walterkozlowski That is fine. I will go with whatever is decided although I must point out that NFVI is well defined and understood widely whereas cloud infrastructure is not. Same for VIM.